### DISCUSSION TITLE
Database si espande all'infinito

### DISCUSSION THREAD

**Discussion by sky7176** [indomus/forum#189] [HUB: Home Assistant]
Ciao a tutti,
dopo l'ultimo aggiornamento di hass.io, il mio db ha iniziado ad espandersi, considerando che tengo 2 giorni di storico, mi aspetto, e così faceva, che dopo due giorni il sistema cancelli i dati più vecchi e faccia spazio per quelli nuovi, di conseguenza una volta raggiunta una certa dimesnione del db questa non dovrebbe più cambiare di molto al netto di inserimento di nuove entità.

Avete notato anche voi questo comportamento?

Vi riporto il pezzo di config.yaml che utilizzo per gestire lo storico, ripeto, sino all'ultimo aggiornamento andava a meraviglia.

```############################################################################
#https://home-assistant.io/components/recorder/
recorder:
  #purge_interval: 1
  purge_keep_days: 2
  commit_interval: 60
  include:
    domains:
    - sensor
    - device_tracker
    #- climate
    #- binary_sensor
    #- light
  exclude:
    entities:
    - sensor.ora_aggiornamento_2
    - sensor.last_boot
    - sensor.date
    - sensor.time
    - sensor.workday
    - sun.sun
    - sensor.moon
    - sensor.fasi_lunari
    - sensor.stagioni
    - sensor.season
    - sensor.ora_aggiornamento
    - sensor.home_assistant_operating_system_version
    - binary_sensor.node_red_update_available
    - binary_sensor.esphome_update_available
    - binary_sensor.file_editor_update_available
    - binary_sensor.home_assistant_google_drive_backup_update_available
    - binary_sensor.samba_share_update_available
    - binary_sensor.sqlite_web_update_available
    - binary_sensor.tasmoadmin_update_available
    - binary_sensor.terminal_ssh_update_available
    - sensor.zigbee2mqtt_bridge_state
    - sensor.coordinator_version
    - input_select.zigbee2mqtt_log_level
    - switch.zigbee2mqtt_main_join
    - timer.zigbee_permit_join

    
history:
  exclude:
    domains:
      - automation
      - weblink
      - updater
      - switch
      - media_player
    entities:
      - sensor.last_boot
      - sensor.date
  include:
    domains:
      - sensor
      - device_tracker
     # - switch
     # - media_player

logbook:
  exclude:
    entities:
      - sensor.last_boot
      - sensor.date
    domains:
      - sun
      - weblink
  include:
    domains:
      - sensor
      - switch
      - media_player
############################################################################
```

**Comment by snakuzzo** [comment]
Sinceramente non ho notato alcuna differenza. Sono su Core ed è sempre sui circa 80MB

**Reply by sky7176** [reply]
Ciao,
ho provato a cancellare il vecchio db e farlo ricreare, dopo 12 ore sono già a 128 MB quando prima per 7 giorni giravo sui 170 MB in totale.
Ho anche controllato se avessi entità "sfuggite" dall'esclusione ma non ho trovato nulla di strano.

Bho indagerò.

Grazie
Giuseppe

**Reply by sky7176** [reply]
Ok forse ho trovato.
Ho un esp8266 tasmota con un sensore di fulmini AS3935 che deve essere impazzito, praticamente mi manda messaggi ogni secondo.

Ora ho provato a spegnere il sensore e voglio vedere se tutto torna normale.
